client/cli/build.rs

```diff
@@ -10,9 +10,9 @@ fn main() {
         } else {
             // we set a fake hash here and make it easily recognizable
             // by giving it a prefix (0x0123456)
-            String::from("0x01234569afbe457afa1d2683a099c7af48a523c")
+            String::from("01234569afbe457afa1d2683a099c7af48a523c1")
```

Comment:
This was wrong. `0x` is not returned by `git rev-parse HEAD` and one character was truncated.

Reply:
I'll keep that in mind for  the other `bins`